src/Components/Shared/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import styles from './button.module.css';
+
+const Button = ({ text, whenClicked, type }) => {
+  return (
+    <button
+      type={type}
+      className={type == 'submit' ? styles.confirm : styles.cancel}
```

Comment:
If we are going to handle the different buttons with the type, when we have a form, we should always execute the onSubmit from the "form" and this could give us some problems.
I recommend you to use another prop to handle the styles, for example:
```suggestion
      className={variant === 'primary' ? styles.confirm : styles.cancel}
```